src/fromager/hooks.py

```diff
@@ -29,7 +29,7 @@ def _die_on_plugin_load_failure(
     mgr: hook.HookManager,
     ep: extension.Extension,
     err: Exception,
-):
+) -> None:
```

Comment:
```suggestion
) -> typing.NoReturn:
```

Reply:
Ah is this to differentiate if we are actually returning None vs not doing any returns?

Reply:
The function is raising an exception, so it never returns anything. https://docs.python.org/3/library/typing.html#typing.NoReturn